src/ape/contracts/base.py

```diff
@@ -842,6 +843,17 @@ def at(self, address: AddressType, txn_hash: Optional[str] = None) -> ContractIn
             address, self.contract_type, txn_hash=txn_hash
         )
 
+    @cached_property
+    def source_path(self) -> Optional[Path]:
+        """
+        Returns the path to the local contract if determined that this container
+        belongs to the active project.
+        """
+        contract_name = self.contract_type.name
+        contract_type = self.project_manager._get_contract(contract_name)
+        if contract_type and self.contract_type:  # == contract_type:
+            return self.project_manager.contracts_folder / self.contract_type.source_id
```

Comment:
`contract_type.source_id` was empty so I used `self.contract_type.source_id`

Reply:
oh okay, so line 853 actually returns a containers, I added a separate comment asking to change var name